packages/sdk/src/config/config.ts

```diff
@@ -16,9 +16,13 @@ export const DEFAULT_CONTRACT_ADDRESS =
   '0x3a4Ab33F3D605e75b6D00A32A0Fa55C3628F6A59'.toLowerCase();
 
 // iExec Dataset Registry
-export const POCO_REGISTRY_CONTRACT_ADDRESS =
+export const POCO_DATASET_REGISTRY_CONTRACT_ADDRESS =
   '0x799daa22654128d0c64d5b79eac9283008158730'.toLowerCase();
 
+// iExec Dataset Registry
```

Comment:
Comment to be updated I guess

Reply:
Commit: 238cf8a10453c058a91bc07c72b722a6f95c8151